From 5ca551a3a14c7ead545b8e8a72d1244b77824aed Mon Sep 17 00:00:00 2001
From: Karsten Tausche <karsten.tausche@student.hpi.uni-potsdam.de>
Date: Tue, 6 Jan 2015 19:18:02 +0100
Subject: [PATCH] vtkImageDataLIC2D

---
 Rendering/LIC/vtkImageDataLIC2D.cxx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Rendering/LIC/vtkImageDataLIC2D.cxx b/Rendering/LIC/vtkImageDataLIC2D.cxx
index c10de4e..cc3bf72 100644
--- a/Rendering/LIC/vtkImageDataLIC2D.cxx
+++ b/Rendering/LIC/vtkImageDataLIC2D.cxx
@@ -574,10 +574,10 @@ int vtkImageDataLIC2D::RequestData(
   int magUpdateExt[6];
   magUpdateExt[2*comp[0]] = updateExt[2*comp[0]] * this->Magnification;
   magUpdateExt[2*comp[1]] = updateExt[2*comp[1]] * this->Magnification;
-  magUpdateExt[2*comp[2]] = updateExt[comp[2]];
+  magUpdateExt[2*comp[2]] = updateExt[2*comp[2]];
   magUpdateExt[2*comp[0]+1] = (updateExt[2*comp[0]+1] + 1) * this->Magnification - 1;
   magUpdateExt[2*comp[1]+1] = (updateExt[2*comp[1]+1] + 1) * this->Magnification - 1;
-  magUpdateExt[2*comp[2]+1] = updateExt[comp[2]];
+  magUpdateExt[2*comp[2]+1] = updateExt[2*comp[2]+1];
 
   vtkPixelExtent magLicExtent(
         magUpdateExt[2*comp[0]],
-- 
1.9.4.msysgit.0

